src/main/java/org/openmaptiles/layers/Transportation.java

```diff
@@ -538,7 +670,8 @@ public List<VectorTile.Feature> postProcess(int zoom, List<VectorTile.Feature> i
       }
     }
 
-    var merged = FeatureMerge.mergeLineStrings(items, minLength, tolerance, BUFFER_SIZE);
+    // "shipway_linestring_gen_z5: ... tolerance: ZRES6", etc. when recalculated from meters to pixels is always 0.5
+    var merged = FeatureMerge.mergeLineStrings(items, minLength, 0.5, BUFFER_SIZE);
```

Comment:
One way that planetiler-openmaptiles does differ from openmaptiles is that pt-omt includes more detailed geometries by default, but you can change this with the `--simplify-tolerance` argument. I noticed when looking at 3.15 vs 3.14 output that roads look "blockier" in 3.15. Could we make this simplify tolerance use the default values from arguments like `Boundary` layer does?

https://github.com/openmaptiles/planetiler-openmaptiles/blob/827673b4ea20eb1a802c34eed4580ef7fe5f8bc3/src/main/java/org/openmaptiles/layers/Boundary.java#L354-L359

Reply:
Thanks!! It might be worth playing with the planetiler settings a bit to see what value the default tolerance gets as close as possible to openmaptiles - we could even make that the default, but it's good for whoever's running it to be able to configure.